vendor/k8s.io/kubernetes/cluster/saltbase/salt/etcd/etcd.manifest

```diff
@@ -32,7 +32,10 @@
 "kind": "Pod",
 "metadata": {
   "name":"etcd-server{{ suffix }}",
-  "namespace": "kube-system"
+  "namespace": "kube-system",
+  "labels": {
+    "k8s-app" : "etcd-server{{ suffix }}"
```

Comment:
Changing this file is a no-go, I'm afraid.  It is in vendor, so we don't own it.  And we better not be using it :-)